src/d3a/models/strategy/area_agents/one_sided_alternative_pricing_agent.py

```diff
@@ -66,8 +68,19 @@ def _buy_energy_alternative_pricing_schemes(self, area):
                                "An Error occurred while buying an offer")
 
     def event_tick(self, *, area):
-        if area.current_tick_in_slot - self.MIN_SLOT_AGE >= 0 and \
+        if area.current_tick_in_slot >= self.MIN_SLOT_AGE and \
                 ConstSettings.IAASettings.AlternativePricing.PRICING_SCHEME != 0:
             self._buy_energy_alternative_pricing_schemes(area)
 
-        super().event_tick(area=area)
+        # super().event_tick(area=area)
```

Comment:
Please remove this or add a comment with reason for keeping it. Thanks!

Reply:
Ah, you are right! Removing it completely, no reason for it to stay.